migrations_lockfile.txt

```diff
@@ -5,12 +5,7 @@ ahead of you.
 To resolve this, rebase against latest master and regenerate your migration. This file
 will then be regenerated, and you should be able to merge without conflicts.
 
-admin: 0002_logentry_remove_auto_add
```

Comment:
The apps ignored are apps that do not file migrations in the source code. I believe these are Django related migrations that we don't have control over. Please let me know if I got it right.